tools/asm-processor/prelude.inc

```diff
@@ -41,3 +41,6 @@
 .set $fs4f,         $f29
 .set $fs5,          $f30
 .set $fs5f,         $f31
+
+# Floating-point status register
+.set $FpcCsr,       $31
```

Comment:
This is a subrepo, right? So this should be PR'd to that repo. And the permuter might need it too. And maybe decomp.me?

Reply:
No. As every time this topic comes up, the asm processor is not a subrepo... for reasons I can never remember.

If using this register alias is an issue I can disable it on rabbitizer's configuration

Reply:
https://github.com/zeldaret/mm/pull/278#pullrequestreview-735244353

Reply:
I think we can probably live without this, I expect mips2c wouldn't like it either.

Reply:
Removed the use of `$FpcCsr` for now instead